FluidNC/src/Maslow/Maslow.cpp

```diff
@@ -1549,7 +1549,7 @@ String Maslow_::axis_id_to_label(int axis_id) {
 //Checks to see if the calibration data needs to be sent again
 void Maslow_::checkCalibrationData() {
     if (calibrationDataWaiting > 0) {
-        if (millis() - calibrationDataWaiting > 30000) {
+        if (millis() - calibrationDataWaiting > 27000) {
```

Comment:
What if we made this not a multiple of 1000 ms?

Reply:
Yes, a true non divisible by 10 makes sense.